lib/rucio/core/meta_conventions.py

```diff
@@ -137,54 +143,64 @@ def add_value(key, value, *, session: "Session"):
 
         raise RucioException(error.args)
 
-    k = session.query(models.DIDKey).filter_by(key=key).one()
+    statement = select(
+        models.DIDMetaConventionsKey.value_regexp,
+        models.DIDMetaConventionsKey.value_type
+    ).where(
+        models.DIDMetaConventionsKey.key == key
+    )
+    query = session.execute(statement).one()
```

Comment:
Also use `.scalars()` here, the `query[0]`, `query[1]` etc is not very nice :-)
See https://github.com/rucio/rucio/blob/2c5b291f53caebf85259010f11791ed7f497572f/lib/rucio/core/rse.py#L1115